### PR TITLE
chore: add `x/wasm` module to `simapp`

### PR DESCRIPTION
**Pull request by zemyblue**
## Description

Add `x/wasm` module to `simapp` of lbm-sdk default app.
And DockerFile also fixed to build cosmwasm library for `x/wasm` module.

closes: #557 

## Motivation and context

default simapp of lbm-sdk doesn't have wasm feature

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [x] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: simapp/simd/cmd/root.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"errors"
+
```
Comment (dudong2) [inline simapp/simd/cmd/root.go:5]: Why did you add empty line here?

**Review by dudong2 — APPROVED** [review]

**Review by loloicci — APPROVED** [review]

**Review by 0Tech — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -14,23 +14,48 @@ FROM golang:alpine AS build-env
...
+RUN go mod download
+
+# Build cosmwasm
+ENV RUSTUP_HOME=/usr/local/rustup
+ENV CARGO_HOME=/usr/local/cargo
+ENV PATH=$CARGO_HOME/bin:$PATH
+
+RUN wget "https://static.rust-lang.org/rustup/dist/x86_64-unknown-linux-musl/rustup-init"
+RUN chmod +x rustup-init
+RUN ./rustup-init -y --no-modify-path --default-toolchain 1.57.0; rm rustup-init
```
[the remark this answers is not in the document]
Comment (zemyblue) [reply Dockerfile:39]: This semicolon was used in order to execute two command as one line.

File: simapp/simd/cmd/root.go (hunk 1 above)
Comment (zemyblue) [reply simapp/simd/cmd/root.go:5]: No meaning. I think this line was added automatically by IDE.
